Week-7/Exercise7.3.js

```diff
@@ -0,0 +1,27 @@
+let arr = [0, 2, 1, 2, 0];
+let count0 = 0, count1 = 0, count2 = 0, N = arr.length;
+
+for(let el of arr){
```

Comment:
solve this problem using Dutch National Flag problem
this is to teach you low, mid and high probelm